### PR TITLE
PostgreSQL - article latest module error

### PR DESCRIPTION
**Pull request by alikon**
#### Steps to reproduce the issue

Set the Articles - Latest module parameter Order  to Random Article

Click on Latest Articles item from the fronted left menu All Modules
#### Actual result

you got an error message
#### Addictional Comments

similar to #8238,#8240,#8242

### CODE REVIEW

File: modules/mod_articles_latest/helper.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ public static function getList(&$params)
 			'mc_dsc' => 'CASE WHEN (a.modified = ' . $db->quote($db->getNullDate()) . ') THEN a.created ELSE a.modified END',
 			'c_dsc' => 'a.created',
 			'p_dsc' => 'a.publish_up',
-			'random' => 'RAND()',
+			'random' => JFactory::getDbo()->getQuery(true)->Rand()',
```
Comment (zero-24) [inline modules/mod_articles_latest/helper.php]: @alikon we need to use here: `'random' => JFactory::getDbo()->getQuery(true)->Rand(),` without the `'` at the end ;)

Else we get: `Parse error: syntax error, unexpected '',' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in JROOT/modules/mod_articles_latest/helper.php on line 106`

Comment (alikon) [inline modules/mod_articles_latest/helper.php]: @zero-24  thx

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on ca3fbb3def3768156d9f218e0610f4bfbbd80ad8<br /><br />Works now :smiley: :+1: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8243">issues.joomla.org/joomla-cms/8243</a>.</sub>

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on ca3fbb3def3768156d9f218e0610f4bfbbd80ad8<br /><br />Thanks alikon!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8243">issues.joomla.org/joomla-cms/8243</a>.</sub>

**Comment by zero-24** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8243">issues.joomla.org/joomla-cms/8243</a>.</sub>

File: modules/mod_articles_latest/helper.php [hunk 2]
```diff
@@ -101,7 +101,7 @@ public static function getList(&$params)
 			'mc_dsc' => 'CASE WHEN (a.modified = ' . $db->quote($db->getNullDate()) . ') THEN a.created ELSE a.modified END',
 			'c_dsc' => 'a.created',
 			'p_dsc' => 'a.publish_up',
-			'random' => 'RAND()',
+			'random' => JFactory::getDbo()->getQuery(true)->Rand(),
```
Comment (roland-d) [inline modules/mod_articles_latest/helper.php]: Why are you getting a new DB object when on line 37 there is already a $db. So you can improve this by doing:
```
$db->getQuery(true)->Rand();

**Comment by alikon** [comment]
modifyied for using already present $db object as per @roland-d  comment

**Pull request merged**